### DISCUSSION TITLE
Table中的Cancel(取消保存)按钮能否增加一个回调

### DISCUSSION THREAD

**Discussion by SamXMG** [dotnetcore/BootstrapBlazor#4878] [Q&A] [answered]
能不能给Table增加一个OnCancelAsync的参数用于在取消保存之后执行某些操作

**Comment by ArgoZhang** [comment, marked as answer]
@SamXMG 

https://github.com/dotnetcore/BootstrapBlazor/issues/4749?wt.mc_id=DT-MVP-5004174

看看这个 Issue 相关的是否满足你的需求

**Reply by SamXMG** [reply]
Thank you!,更新到最新版本后已解决这个问题
